src/contact.js

```diff
@@ -0,0 +1,40 @@
+const contentInfo = {
+  srcLink1: "../src/assests/contact-img.jpg",
+  srcLink2: "../src/assests/map-img.jpg",
+  srcLink3: "../src/assests/email-img.png",
+  srcLink4: "../src/assests/phone-img.png",
+  paraText1: "1. Location ( Kanpur )",
+  paraText2: "2. Email ( ahmadiklaq3@gmail.com)",
+  paraText3: "3. Phone ( +916307329321 )",
+};
+
+function addImageElement(contentDiv, srcLink) {
```

Comment:
Move this utility functions to a new file called `utils.js` and import these functions here from `utils.js`